src/transformers/image_transforms.py

```diff
@@ -303,12 +303,14 @@ def normalize(
             raise ValueError(f"mean must have {num_channels} elements if it is an iterable, got {len(mean)}")
     else:
         mean = [mean] * num_channels
+    mean = np.array(mean, dtype=image.dtype)
```

Comment:
This change is to prevent type changes happening. Previously `float32` arrays were cast to `float64`